src/daemon/compilation_daemon.h

```diff
@@ -51,6 +50,11 @@ class CompilationDaemon : public BaseDaemon {
                          const cache::ExtraFiles& extra_files,
                          const cache::FileCache::Entry& entry);
 
+  bool Check(const Configuration& conf) const override;
+  inline bool Reload(const Configuration& conf) override {
```

Comment:
For the purpose of uniformity of chained calls inside further overridden methods. Since I can't just invoke "the first defined same method in the nearest base class".